crates/bevy_gltf/Cargo.toml

```diff
@@ -30,6 +33,9 @@ bevy_utils = { path = "../bevy_utils", version = "0.12.0-dev" }
 # other
 gltf = { version = "1.0.0", default-features = false, features = [
     "KHR_lights_punctual",
+    "KHR_materials_transmission",
+    "KHR_materials_ior",
+    "KHR_materials_volume",
```

Comment:
should those features be enabled only if `pbr_transmission_textures` is enabled?

Reply:
`pbr_transmission_textures` allows us to use textures for each of these parameters, but we still need those flags to read the uniform/scalar values that modulate the whole effect.